packages/react-divider/src/components/Divider/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './Divider';
-export * from './Divider.types';
+// Explicit exports to ommit CheckboxCommon
```

Comment:
Seems like a copy-paste error happened here
```suggestion
// Explicit exports to omit DividerCommons
```